Adyen/Core/Core Protocols/PresentableComponent.swift

```diff
@@ -62,15 +62,26 @@ public extension PresentableComponent {
 
 @_spi(AdyenInternal)
 public protocol TrackableComponent: Component {
-
-    func sendTelemetryEvent()
+    
+    func fetchCheckoutAttemptId()
```

Comment:
The naming feels a bit misleading as it's actually sending analytics events right?
Fetching the `checkoutAttemptId` is only a sideeffect isn't it?